transformers/events/jug_file/base/transformer.go

```diff
@@ -37,17 +37,24 @@ func (Transformer) ToModels(_ string, logs []core.EventLog, _ *postgres.DB) ([]e
 		what := shared.DecodeHexToText(log.Log.Topics[2].Hex())
 		data := shared.ConvertUint256HexToBigInt(log.Log.Topics[3].Hex())
 
+		msgSender := shared.GetChecksumAddressString(log.Log.Topics[1].Hex())
```

Comment:
Why do you need GetCheckSumAddressString? I don't think I've seen this elsewhere.

Reply:
I just added this as a helper so avoid converting the topic hex to an address, and then back to hex again, eg: 
 `common.HexToAddress(log.Log.Topics[1].Hex()).Hex()`.

Not sure if this is a patter we want to use, but thought I'd see how it looked. Happy to change it back if that makes more sense.